Append std:: to call to setiosflags to make it build with libc++.
https://crbug.com/747600
diff -Nuar old/ragel/javacodegen.cpp new/ragel/javacodegen.cpp
--- old/ragel/javacodegen.cpp
+++ new/ragel/javacodegen.cpp
@@ -1184,7 +1184,7 @@
 {
 	item_count++;
 
-	out << setw(5) << setiosflags(ios::right) << item;
+	out << setw(5) << std::setiosflags(ios::right) << item;
 	
 	if ( !last ) {
 		if ( item_count % SAIIC == 0 ) {
